src/apps/activity_assignments/errors.py

```diff
@@ -4,4 +4,16 @@
 
 
 class ActivityAssignmentActivityOrFlowError(ValidationError):
```

Comment:
I'd suggest renaming this to the below to more accurately describe its purpose:
```suggestion
class ActivityAssignmentBothActivityAndFlowError(ValidationError):
```